apps/core/app/[locale]/(default)/account/change-password/_actions/submit-customer-change-password-form.ts

```diff
@@ -0,0 +1,56 @@
+'use server';
+
+import { z } from 'zod';
+
+import { CustomerChangePasswordSchema } from '~/client/mutations/submit-change-password';
+import { submitCustomerChangePassword } from '~/client/mutations/submit-customer-change-password';
+
+export interface State {
+  status: 'idle' | 'error' | 'success';
+  message?: string;
+}
+
+export const submitCustomerChangePasswordForm = async (
+  _previousState: unknown,
+  formData: FormData,
+) => {
+  try {
+    const parsedData = CustomerChangePasswordSchema.parse({
+      newPassword: formData.get('new-password'),
+      currentPassword: formData.get('current-password'),
+      confirmPassword: formData.get('confirm-password'),
+    });
+
+    const response = await submitCustomerChangePassword({
+      newPassword: parsedData.newPassword,
+      currentPassword: parsedData.currentPassword,
+    });
+
+    if (response.errors.length === 0) {
+      return { status: 'success', message: '' };
```

Comment:
Should we return `response` here?

Reply:
I've added a message here but we basically don't use  any data.